qt-core/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Change Log
 
+## 1.5.0 (Apr 10, 2025)
+
+🎉 **Added**
+
+- Ignoring of CMake presets to Qt recommended settings
```

Comment:
What does that mean?

Reply:
I changed it with https://github.com/qt-labs/vscodeext/pull/91/commits/f49c0b9e83ce13366d4d2bcfdce71f7a323adb93